test/mocking_classes.py

```diff
@@ -133,7 +131,7 @@ def __init__(
     def maxstep(self):
         return 100
 
-    def _set_seed(self, seed: Optional[int]):
+    def _set_seed(self, seed: int | None):
```

Comment:
I think this type notation was introduced in py3.10. Do you still targeting 3.9 ?

Reply:
there should be a `from __future__ import annotations` in each file - maybe not this one!